.azure-pipelines-templates/azure_cli.yml

```diff
@@ -0,0 +1,11 @@
+steps:
+  - script: |
+      set -ex
+      curl -sL https://aka.ms/InstallAzureCLIDeb | sudo bash
+      az login --service-principal -u $(CCF_SNP_CI_APP_ID) -p $(CCF_SNP_CI_SERVICE_PRINCIPAL_PASSWORD) --tenant $(CCF_SNP_CI_TENANT)
+      pip install azure-mgmt-resource azure-identity azure-mgmt-containerinstance
+    name: setup_azure_cli
```

Comment:
It'd be nice to add a `displayName` for these steps so that the UI is consistent with other jobs, e.g. "Setup Azure CLI`.

Reply:
https://github.com/microsoft/CCF/pull/4662/commits/1c052bf6e28d0ec2033795cdeff8ec6b0a172432